README.md

```diff
@@ -146,6 +146,31 @@ const healthCardFromQR = await reader.fromQRNumeric(qrNumericStrings);
 console.log('Bundle from QR:', await healthCardFromQR.asBundle());
 ```
 
+#### Usage with the VCI Directory Snapshot
+
+The library can optionally consult the VCI directory snapshot to obtain a canonical collection of issuer metadata (JWKS and CRLs). This is useful when you want an authoritative, ready-made source of issuer keys without assembling or maintaining your own `Directory` object. In order to bundle it directly into your `SHCReader` object, you may provide the `useVciDirectory: true` parameter to its constructor.
```

Comment:
Simplify this description. Explain what VCI is. Say that you can use this to let the library automatically fetch public keys, revocation lists (CRLs) and other metadata from known SHC issuers.